doc/Language/list.pod6

```diff
@@ -480,14 +480,17 @@ which is selecting the 4 to 6th element from the three  first dimensions (C<^3>)
 A L<C<Range>|/type/Range> is a container for a lower and an upper boundary.
 Generating a slice with a C<Range> will include any index between those bounds,
 including the bounds. For infinite upper boundaries we agree with
-mathematicians that C<Inf> equals C<Inf-1>.
+mathematicians that C<Inf> equals C<Inf-1>. Slicing with an infinite Range
+will L<truncate|/language/subscripts#Truncating_slices>.
 
     my @a = 1..5;
     say @a[0..2];     # OUTPUT: «(1 2 3)␤»
     say @a[0..^2];    # OUTPUT: «(1 2)␤»
     say @a[0..*];     # OUTPUT: «(1 2 3 4 5)␤»
-    say @a[0..^*];    # OUTPUT: «(1 2 3 4)␤»
+    say @a[0..^*];    # OUTPUT: «(1 2 3 4 5)␤»
     say @a[0..Inf-1]; # OUTPUT: «(1 2 3 4 5)␤»
+    say @a[0..*-1];   # OUTPUT: «(1 2 3 4 5)␤»
+    say @a[0..^*-1];  # OUTPUT: «(1 2 3 4)␤»
```

Comment:
Infinite minus one minus one is still infinite. How does this apply here?

Reply:
It doesn't apply. `1..*` is the same as `1..Inf` with or without the caret. But `1..*-1` suddenly is a WhateverCode that generates a Range whose upper boundary is filled in by the subscript operator with the list's .elems. So subtracting 1 twice indeed loses the last value. The interesting point is that the infinite ranges `1..*` and `1..^*` behave as if they were also WhateverCode, making the use of `*` strangely consistent. I take it that this (up to the strangely consistent part) should be explained on the page, too. Correct?

Reply:
Yep, probably. Clarifying that would really be helpful.